hooks/pre_gen_project.py

```diff
@@ -1,23 +1,46 @@
 import re
 import sys
+from pathlib import Path
 
-PROJECT_REGEX = r'^[-a-zA-Z][-a-zA-Z0-9]+$'
-project_slug = '{{ cookiecutter.project_slug }}'
 
-if not re.match(PROJECT_REGEX, project_slug):
-    print('ERROR: %s is not a valid project slug. Slug can only contain letters, digits, '
-          'and dashes.' % project_slug)
+PROJECT_DIR = "{{ cookiecutter.project_dir }}"
+
+FULL_PROJECT_DIR = (Path(".") / PROJECT_DIR).resolve()
+FORBIDDEN_CHARS = r'<>:"\/|?*'
+
+forbidden_chars_in_dir_name = any(ch in PROJECT_DIR for ch in FORBIDDEN_CHARS)
```

Comment:
just an idea: `... = set(PROJECT_DIR) & set(FORBIDDEN_CHARS)` - may be easier to read. your solution is fine too, feel free to keep it

Reply:
wow, that's a cool hack! thank you for the hint, applied in c7d7b05